packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -31,6 +32,7 @@ export interface AbstractMessage {
   type: string;
   rawSig?: string;
   securityProviderResponse?: Record<string, Json>;
+  securityAlertResponse?: Record<string, Json>;
```

Comment:
Is this name distinct enough from `securityProviderResponse`?

Would there be value in being more explicit and calling the new one `ppomResponse` perhaps or even combining both values in a single `securityProviderResponse` object?

Reply:
@matthewwalsh0 : opensea is going to be deprecated as we introduce PPOM. Thus eventually we will remove `securityProviderResponse`.

A approach could be we use `securityProviderResponse` for PPOM validation response also.

Reply:
We do not use `ppomResponse` as in future we may have more security providers.

Reply:
Understood, if we're removing OpenSea, then this temporary duplication should be harmless as in future we can just have the `securityAlertResponse`.